### ISSUE TITLE
Chart - On navigating to Graphs NVDA should announce the instructions as hit right arrow key to navigate to other section for further information.

### ISSUE THREAD

**Issue by LucyLiYY** [microsoft/fluentui#19176]
### Environment Information

The issue repros on Fluent UI Charting Example site:
https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/charting/dist/index.html#/examples/stackedbarchart

### Describe the issue:

On navigating to Graphs, NVDA is  not announcing any instruction on how to navigate to next section.
![image](https://user-images.githubusercontent.com/17995942/127445748-d82fc1bb-904c-4857-bbc9-66609b273c55.png)

#### Actual behavior:

On navigating to Graphs, NVDA is announcing inappropriate information. It just says, “Clickable first 40 graphic”. It is not announcing any instruction on navigation.

#### Expected behavior:

On navigating to Graphs, NVDA should announce the instructions as "Activate Focus Mode, click right arrow key to navigate to next section for further information" etc. It should announce as “Clickable first 40 graphic, Activate Focus Mode, click right arrow key to navigate to next section” Like on Hover through mouse, the data which is conveyed to user should be notified by screen reader as well.

**Comment by ling1726** [comment]
@Raghurk  can you take a look at this ?

**Comment by LucyLiYY** [comment]
Is it by design?

**Comment by LucyLiYY** [comment]
@Raghurk any updates?
